recipes/topologytoolkit/meta.yaml

```diff
@@ -47,6 +47,7 @@ test:
 
 about:
   home: https://topology-tool-kit.github.io/
+  license: OTHER
```

Comment:
BSD. btw, in principle, the license should be shipped somehow with the conda-forge binary distribution.

Reply:
What variant of BSD is this? 3-Clause BSD?
Btw, the file specified in `licence_file`, i.e. LICENSE from the ttk source, will be added to the package.

Reply:
> it's a variant of the original 4-clause BSD.
> if it's too exotic we can leave it at "other" for the moment.

@scopatz What would be the correct way to specify `license`/`license_family` in this case?

Reply:
looks good to me.